backend/inspirehep/submissions/marshmallow/seminars.py

```diff
@@ -82,6 +89,13 @@ def convert_to_form_data(self, data):
         end_datetime = data.get("end_datetime")
         form_end_datetime = iso_utc_to_local_form_datetime(end_datetime, timezone)
 
+        literature_records = missing
+        if "literature_records" in data:
+            literature_records = [
+                PidStoreBase.get_pid_from_record_uri(rec["record"]["$ref"])[1]
+                for rec in data.get("literature_records")
```

Comment:
```suggestion
                for rec in data.get("literature_records", [])
```
and you could drop the if